internal/handlers/common/error.go

```diff
@@ -161,4 +261,6 @@ func formatBitwiseOperatorErr(err error, operator string, maskValue any) error {
 // check interfaces
 var (
 	_ error = (*Error)(nil)
+	_ error = (*WriteError)(nil)
```

Comment:
Is there an API that could return a single write error of this type? Should both `WriteError` and `WriteErrors` implement error interface?

And we should check that Error/CommandError and WriteErrors implement ProtoErr interface

Reply:
No, `WriteError` always returned as a part of `WriteErrors`. I made `WriteError` unexported.